src/components/List_POC/Title.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+const Title = styled.span`
```

Comment:
You might want to consider extending the ‘Text’ component here

Reply:
👍